commit 8058eab4bde5e9f745458bbda96aac14686a0145
Author: Michael Still <mikal@stillhq.com>
Date:   Mon Sep 21 17:00:07 2026 +1000

    Convert the networking services to JSON logging.
    
    Neutron, Octavia and Designate now honour openstack_logging_json.
    
    Change-Id: I994da1469296312de61ab7a5a3899666a028aa7e
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/ansible/roles/designate/templates/designate.conf.j2 b/ansible/roles/designate/templates/designate.conf.j2
index f68c0b2e9..a9ecb1a25 100644
--- a/ansible/roles/designate/templates/designate.conf.j2
+++ b/ansible/roles/designate/templates/designate.conf.j2
@@ -1,6 +1,7 @@
 [DEFAULT]
 
 debug = {{ designate_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/designate
 {% if service_name == 'designate-api' %}
diff --git a/ansible/roles/neutron/templates/neutron.conf.j2 b/ansible/roles/neutron/templates/neutron.conf.j2
index 094e7b1ad..93f1de99a 100644
--- a/ansible/roles/neutron/templates/neutron.conf.j2
+++ b/ansible/roles/neutron/templates/neutron.conf.j2
@@ -1,6 +1,7 @@
 # neutron.conf
 [DEFAULT]
 debug = {{ neutron_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 {% if service_name == "neutron-server" %}
 log_file = /var/log/kolla/neutron/neutron-server.log
diff --git a/ansible/roles/octavia/templates/octavia.conf.j2 b/ansible/roles/octavia/templates/octavia.conf.j2
index 8e4050692..6e09f5d8a 100644
--- a/ansible/roles/octavia/templates/octavia.conf.j2
+++ b/ansible/roles/octavia/templates/octavia.conf.j2
@@ -1,6 +1,7 @@
 #jinja2: trim_blocks:False
 [DEFAULT]
 debug = {{ octavia_logging_debug }}
+use_json = {{ openstack_logging_json }}
 
 log_dir = /var/log/kolla/octavia
 {% if service_name == "octavia-api" %}
